### PR TITLE
put rough layout for job details

### PR DESCRIPTION
**Pull request by JDong3**
hi

### CODE REVIEW

File: src/components/JobDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  title: 'VP of Marketing',
+  updatedAt: '2020-07-19T19:04:41.199761Z',
+};
+
+const JobDetails: React.FC = () => {
+  const toBullets = (item: string) => (
+    <li>{item}</li>
+  );
+
+  const top = (
```
Comment (H-Richard) [inline src/components/JobDetails/index.tsx:66]: should be typed React.ReactNode

File: src/components/JobDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      </Grid>
+      <Grid item xs={3}>
+        <Grid container justify="flex-end">
+          <Button variant="contained" color="primary" size="large">Apply For This Job</Button>
+        </Grid>
+      </Grid>
+    </Grid>
+  );
+
+  const bot = (
```
Comment (H-Richard) [inline src/components/JobDetails/index.tsx:81]: perhaps name these something better too

File: src/components/JobDetails/index.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    <Grid container justify="center">
+      <Grid item xs={10}>
+        <br />
+        <Typography variant="h6">Job Description:</Typography>
```
Comment (H-Richard) [inline src/components/JobDetails/index.tsx:85]: Dont need to clarify that its the description

File: src/components/JobDetails/index.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    'Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook ',
+    'Instagram Instagram Instagram Instagram Instagram Instagram Instagram Instagram ',
+    'Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook ',
+  ],
+  title: 'VP of Marketing',
+  updatedAt: '2020-07-19T19:04:41.199761Z',
+};
+
+const JobDetails: React.FC = () => {
+  const toBullets = (item: string) => (
```
Comment (H-Richard) [inline src/components/JobDetails/index.tsx:62]: this doesn't need to be its own component

File: src/components/JobDetails/index.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    title: string,
+    author: User
+
+    createdAt: Date,
+    expiresAt: Date,
+    active: boolean,
+
+    description: string,
+    requirements: Array<string>,
+    desirements: Array<string>,
```
Comment (H-Richard) [inline src/components/JobDetails/index.tsx:23]:
```suggestion
    requirements: string[],
    desirements: string[],
```

less characters

File: src/components/JobDetails/index.tsx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  requirements: [
+    'Instagram Instagram Instagram Instagram Instagram Instagram Instagram Instagram ',
+    'Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook ',
+    'Instagram Instagram Instagram Instagram Instagram Instagram Instagram Instagram ',
+    'Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook ',
+    'Instagram Instagram Instagram Instagram Instagram Instagram Instagram Instagram ',
+    'Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook Facebook ',
+  ],
+  title: 'VP of Marketing',
+  updatedAt: '2020-07-19T19:04:41.199761Z',
```
Comment (H-Richard) [inline src/components/JobDetails/index.tsx:58]:
```suggestion
  updatedAt: new Date('2020-07-19T19:04:41.199761Z'),
```

File: src/components/JobDetails/index.tsx [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    'Skills Skills Skills Skills Skills Skills Skills Skills ',
+    'Youtube Youtube Youtube Youtube Youtube Youtube Youtube Youtube ',
+    'Skills Skills Skills Skills Skills Skills Skills Skills ',
+    'Youtube Youtube Youtube Youtube Youtube Youtube Youtube Youtube ',
+    'Skills Skills Skills Skills Skills Skills Skills Skills ',
+    'Youtube Youtube Youtube Youtube Youtube Youtube Youtube Youtube ',
+    'Skills Skills Skills Skills Skills Skills Skills Skills ',
+    'Youtube Youtube Youtube Youtube Youtube Youtube Youtube Youtube ',
+  ],
+  expiresAt: '2020-07-19T19:04:41.199659Z',
```
Comment (H-Richard) [inline src/components/JobDetails/index.tsx:47]:
```suggestion
  expiresAt: new Date('2020-07-19T19:04:41.199659Z'),
```

File: src/components/JobDetails/index.tsx [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+  }
+};
+const stuff = {
+  active: 'true',
+  author: {
+    email: 'executive@gmail.com',
+    executive: true,
+    firstName: 'Executive',
+    lastName: 'Doe',
+    updatedAt: '2020-07-19T19:04:41.192603Z',
```
Comment (ivinable2000) [inline src/components/JobDetails/index.tsx:33]:
```suggestion
    updatedAt: new Date('2020-07-19T19:04:41.192603Z'),
```

**Pull request merged**